ci-operator/step-registry/openshift/e2e-azure-ovn/openshift-e2e-azure-ovn-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
```

Comment:
look at https://github.com/openshift/release/pull/9612, i think 9612 is much better way to organize these steps. I would try to use that organization.